tests/requirements.txt

```diff
@@ -1,2 +1,4 @@
+alembic >= 1.5.2
 gabbi >= 2.1.0
+psycopg2 >= 2.8.6
```

Comment:
psycopg2-binary to speed it up? :D

Reply:
Oh, shoot, I forgot about that package again! :) Will do!